scripts/prepare-generate.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env sh
 
-GO111MODULE=off go get -u k8s.io/code-generator/cmd/deepcopy-gen
-GO111MODULE=off go get -u github.com/golang/protobuf/protoc-gen-go
-GO111MODULE=off go get -u golang.org/x/tools/cmd/stringer
-
+tmp="$(mktemp -d)"
+cd "${tmp}" || exit 1
+go get k8s.io/code-generator/cmd/deepcopy-gen@v0.17.2
+go get github.com/golang/protobuf/protoc-gen-go@v1.3.3
+go get golang.org/x/tools/cmd/stringer@v0.0.0-20200130002326-2f3ba24bd6e7
+rm -rf "${tmp}"
```

Comment:
just a note: 'rm -rf $PWD'  works too, no need for tmp variable
also, 'go get' can accept package list to fetch

Reply:
I think this variant is more readable.